core/src/main/res/layout/item_help.xml

```diff
@@ -21,7 +21,7 @@
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:contentDescription="@string/expand"
-    android:tint="@color/icons"
+    android:tint="@color/grey"
```

Comment:
Rather changing the tint color in`core/src/main/res/layout/item_help.xml`
Would you please try adding a line in `core/src/main/res/values-night/color.xml` like 
`<color name="icons">@color/grey</color>` which will make the arrow perfecty visible in both the modes.

Reply:
I did not do that because of the discussion at the issue.

> I really wonder why this does not work out of the box here! It should be grey like the text to me.

Since it has to grey to match the text in both dark and light mode, I think it's fine this way.